CMakeLists.txt

```
@@ -1146,6 +1146,9 @@ ELSEIF(ENABLE_NCZARR)
 ENDIF()
 IF (ENABLE_FILTER_BZ2)
   FIND_PACKAGE(Bz2)
+  IF(NOT Bz2_FOUND)
+    FIND_PACKAGE(BZip2)
+  ENDIF(NOT Bz2_FOUND)
```

"Bz2" exists neither in this repo (after this PR) nor in CMake nor in Bzip2 (as CMake config).
```suggestion
  FIND_PACKAGE(BZip2)
```